crates/compiler/mono/src/layout.rs

```diff
@@ -3251,9 +3251,17 @@ pub fn ext_var_is_empty_tag_union(subs: &Subs, ext_var: Variable) -> bool {
     // the ext_var is empty
     let mut ext_fields = std::vec::Vec::new();
     match roc_types::pretty_print::chase_ext_tag_union(subs, ext_var, &mut ext_fields) {
-        Ok(()) | Err((_, Content::FlexVar(_) | Content::RigidVar(_) | Content::Error)) => {
-            ext_fields.is_empty()
-        }
+        Ok(())
+        | Err((
```

Comment:
`chase_ext_tag_union` should maybe return a custom type? (not important now but this reads kind of weird)